src/ext/resources/gtfsgraphqlapi/schema.graphqls

```diff
@@ -4038,6 +4441,38 @@ enum VertexType {
     PARKANDRIDE
 }
 
+input InputVehicleWalkPreferences {
+    """Speed when walking a vehicle."""
+    speed: Speed
+
+    """
+    Reluctance in a format of `30s + 2.0 x` where the static cost is applied
+    both when getting off the vehicle and when getting on the vehicle again.
+    However, these static costs are not applied when getting on a rented vehicle
+    for the first time or off the vehicle when returning the vehicle. The multiplier
+    is for how bad walking the vehicle is compared to being in transit for equal
+    lengths of time.
+    """
+    reluctance: CostFunction
+
+    """"
+    How long it takes to hop on or off a vehicle when switching to walking the vehicle
+    or when getting on the vehicle again. However, this is not applied when getting
+    on a rented vehicle for the first time or off the vehicle when returning the vehicle.
+    """
+    hopOnOrOffTimeSeconds: Int
```

Comment:
I renamed this from switch time to hopOnOrOff and added seconds postfix but alternatively could have used some PositiveWholeSeconds scalar